docs/src/main/sphinx/release/release-394.md

```diff
@@ -26,16 +28,61 @@
   empty. ({issue}`13785`)
 * Fix failure when reading Parquet data that contains only null values. ({issue}`9424`)
 
+## Druid connector
+
+* Improve query performance when a `varchar` column is interpreted as containing
+  date values in the `uuuu-MM-dd` format and is filtered on. ({issue}`12925`)
```

Comment:
What's `uuuu`? Why the difference in case for `uuuu` and `dd` vs `MM`?

Also, the grammar for this sentence is off. I would rephrase as:
> Improve performance of queries that perform filtering on varchar columns that contain temporal data with the format `YYYY-MM-DD`

Reply:
I think there's some semantic difference between YYYY and uuuu because uuuu goes negative for dates before the year 0 and handles epoch start time differently? @findepi can you clarify if this absolutely needs to be `uuuu`, because otherwise Martin's proposal would be accurate and more user-friendly.

Reply:
I don't think it matters. The point is to describe to users what the general shape is. No one is going to know what `uuuu` means (where is that defined?).

Reply:
`YYYY-MM-DD` works from informal perspective.